FirebaseMLModelDownloader/Sources/ModelFileManager.swift

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+/// Manager for common file operations.
+enum ModelFileManager {
+  /// Root directory of model file storage on device.
+  static var modelsDirectory: URL {
+    // TODO: Reconsider force unwrapping.
+    #if os(tvOS)
+      return FileManager.default.urls(for: .cachesDirectory, in: .userDomainMask).first!
```

Comment:
In the future we will need to make sure we are ready that the model file may be removed by the system and the model path becomes unavailable. Please ignore if it's already handled.

Reply:
Yes, we will have to handle this case. I planned to use `isFileReachable` for this scenario.